tilt-provider.json

```diff
@@ -1,6 +1,7 @@
 {
   "name": "vsphere",
   "config": {
+    "version": "v1.8.99",
```

Comment:
Please also check what the version in clusterctl-setting.json should now be. I think we documented v1.9.99